images/apko/README.md

```diff
@@ -4,8 +4,10 @@
 | - | - |
 | **Status** | stable |
 | **OCI Reference** | `cgr.dev/chainguard/apko` |
-| **Variants/Tags** | ![](https://storage.googleapis.com/chainguard-images-build-outputs/summary/apko.svg) |
 
+
+* [View Image in Chainguard Academy](https://edu.chainguard.dev/chainguard/chainguard-images/reference/apko/overview/)
+* [View Image Catalog](https://console.enforce.dev/images/catalog) for a full list of available tags.
```

Comment:
@pdeslaur do we have a way to deep link to a particular image yet?

Reply:
We don't :( 

Garry's working on a promising new UI, but that won't land before Q3